doc/whatsnew/fragments/7690.new_check

```diff
@@ -0,0 +1,3 @@
+* Extended ``use-dict-literal`` to cover calls to ``dict()`` that could be replaced with a literal dict.
```

Comment:
```suggestion
Extended ``use-dict-literal`` to cover calls to ``dict()`` that could be replaced with a literal dict.
```

Could be the issue with the readthedocs failure since the output of that gives:
`WARNING: Bullet list ends without a blank line; unexpected unindent.`

Reply:
Yes, the news fragments should not contain bullet points, as `towncrier` creates the bullet points for each entry itself.